src/graph/graph.c

```diff
@@ -174,6 +177,12 @@ size_t _Graph_EdgeCap(const Graph *g) {
 	return g->edges->itemCap;
 }
 
+static inline void _Graph_UpdateMatrixDimensions(Graph *g, size_t node_count) {
+	bool build_with_overhead = Config_GetNodeCreationBuffer();
+	if(build_with_overhead) node_count += MAX(node_count * 0.001, MIN_MATRIX_BUFFER);
```

Comment:
Eventually we will have to improve this formula, but this should be good enough for benchmarking, consider changing `0.001` to `0.01`